ci/jenkins/test.sh

```diff
@@ -989,6 +1017,10 @@ if [[ $TESTCASE =~ "multicast" ]]; then
 fi
 echo $MANIFEST_ARGS
 
+if [[ $TESTBED_TYPE == "kind-flexible-ipam" ]]; then
+    MANIFEST_ARGS="$MANIFEST_ARGS --flexible-ipam --multicast --verbose-log"
```

Comment:
It should share code with `if [[ $TESTBED_TYPE == "flexible-ipam" ]]` at L1011.

But I'm confused why we generate manifest in two places: here and kind-setup.sh. Which one is used exactly? One of them must be useless change.

Reply:
it's using manifest generated by test.sh , refactored it.